### ISSUE TITLE
Fulcrum Jekyll - HTML doc needs lang defined

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#612]
The <`html>` element does not have a lang attribute
`.wf-effra-n4-active.wf-effra-i4-active.wf-effra-n7-active.wf-effra-i7-active.wf-effra-n3-active.wf-effra-n5-active.wf-effra-i5-active.wf-active`
`<html class="wf-effra-n4-active wf-effra-i4-active wf-effra-n7-active wf-effra-i7-active wf-effra-n3-active wf-effra-n5-active wf-effra-i5-active wf-active">`
